dashboard/client/src/common/constants.ts

```diff
@@ -1 +1,3 @@
 export const API_REFRESH_INTERVAL_MS = 4000;
+// Per job page refresh interval.
+export const PER_JOB_PAGE_REFRESH_INTERVAL_MS = 10000;
```

Comment:
4 seconds are too short and I don't think it is that useful for actor/task/pg tables. We can reduce the frequency later when we optimize tables (pagination)